# gains of surface normal estimator
gamma_n: 250000.0
beta_n: 150.0

# initial surface normal (in base frame)
initial_n: [0.35, 0.35, 0.866]

# update frequency of surface normal estimator [Hz]
sne_update_frequency: 140.0